Packs/SANS/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Brute Force Investigation - Generic - SANS
+- Switch usage of deprecated **Isolate Endpoint - Generic** playbook with the **Isolate Endpoint - Generic V2** playbook.
```

Comment:
`Changed the deprecated **Isolate Endpoint - Generic** playbook with the **Isolate Endpoint - Generic V2** playbook.`